libr/anal/fcn.c

```diff
@@ -907,7 +908,8 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 		return R_ANAL_RET_ERROR; // MUST BE TOO DEEP
 	}
 
-	if (r_anal_get_fcn_at (anal, addr, 0)) {
+	fcn_at_addr = r_anal_get_fcn_at (anal, addr, 0);
```

Comment:
Define and assign in the same line, same for bbuf

Reply:
ok 6dcfed1ae1f16c9d345bdc36975b341c1b6bb04d